ark/utils/test_utils.py

```diff
@@ -640,7 +640,11 @@ def _make_threshold_mat(in_utils):
     thresh.iloc[:, 1] = .5
 
     if not in_utils:
-        thresh.iloc[:, 0] = np.arange(20) + 2
+        # print(len(np.concatenate([np.arange(2, 14), np.arange(15, 23)])))
```

Comment:
So this function currently isn't used for anything, but will be in the future?

Reply:
Forgot to remove the print statement, sorry.

If you're talking about `_make_threshold_mat` as a whole, it is used in `spatial_analysis_test.py` and `spatial_analysis_utils_test.py`.